collectors/tidpost

```diff
@@ -40,21 +41,20 @@ class TidmarshReceiver:
         return self.subscriber.recv()
 
 
-# the sensors don't self-describe their units, so we map them here. Note that
-# the charge_flags are unpacked
+# the sensors don't self-describe their units, so we map them here. (Legacy
+# protocol only; this is now deprecated as in v2 sensors *do* self-describe
+# their units)
 TIDMARSH_UNITS = {
     "sht_humidity": "percent",
     "bmp_pressure": "hPa",
     "battery_voltage": "volts",
     "illuminance": "lux",
-    "sht_temperature": "celsius",
-    "bmp_temperature": "celsius",
-    "charge_flags_fault": "boolean",
-    "charge_flags_charge": "boolean",
-    "accel_x": "m/s^2",
-    "accel_y": "m/s^2",
-    "accel_z": "m/s^2",
-    "analog": "ADC units"
+    "sht_temperature": "°C",
+    "bmp_temperature": "°C",
+    "accel_x": "g",
+    "accel_y": "g",
+    "accel_z": "g",
+    "analog": "LSB"
 }
 
```

Comment:
Yes, I'm aware of that.  It's unlikely that new sensors will ever be created again that will need to fetch units from that list, but m/s² was never correct for anything that would have been posted from the network (accelerometers have always reported in earth gravity) and "20 celsius" is not really stylistically correct.  "Degrees celsius" would have been fine but verbose, and I don't really see any downside to using the Unicode degree symbol (this is how the sensors will report themselves in the new protocol).  

So yeah, this change likely won't ever affect anything, but I figured since I was making changes I might as well correct it anyway.